# config_data:
#   move_group: "sia5"
#   fixed_frame: "base_link"
#   end_effector: "eef"
#   joint_trajectory_command: "joint_trajectory_action"
#   x_pos: 0.4
#   y_pos: 0.0
#   z_pos: 0.04

config_data:
 move_group: "manipulator"
 fixed_frame: "/crib_steel_base"
 end_effector: "ee_link"
 joint_trajectory_command: "follow_joint_trajectory"
 x_pos: -0.4
 y_pos: -0.2
 z_pos: 0.45
 joint_names: ["shoulder_pan_joint", "shoulder_lift_joint", "elbow_joint", "wrist_1_joint", "wrist_2_joint", "wrist_3_joint"]
